include/odp/api/timer_types.h

```diff
@@ -0,0 +1,32 @@
+/* Copyright (c) 2021, Nokia
+ * All rights reserved.
+ *
+ * SPDX-License-Identifier:     BSD-3-Clause
+ */
+
+/**
+ * @file
+ *
+ * ODP timer service
+ */
+
+#ifndef ODP_API_TIMER_TYPES_H_
+#define ODP_API_TIMER_TYPES_H_
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include <odp/api/std_types.h>
+#include <odp/api/abi/event.h>
+#include <odp/api/abi/pool.h>
+#include <odp/api/abi/queue_types.h>
```

Comment:
Could we drop these includes from this level? It would be easier to maintain the required includes in the headers were they are actually required.

Reply:
Fixed in v2